src/Element/Request/ElementPathParameter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Element\Request;
+
+/**
+ * @internal
+ */
+final readonly class ElementPathParameter
+{
+    public function __construct(
+        private string $elementPath
```

Comment:
Could we just name it path?
Wdyt? It is a little redundant in the moment. :)

Reply:
sure, we can rename it to path :)